webrtc.html

```diff
@@ -4894,6 +4894,7 @@ <h2 id="sec.cert-mgmt">Certificate Management</h2>
              RTCRtcpParameters                         rtcp;
              sequence&lt;RTCRtpCodecParameters&gt;           codecs;
              RTCDegradationPreference                  degradationPreference = "balanced";
+             boolean                                   upscaleAllowed = false;
```

Comment:
Hmm - can you point me at the notes on this because I had in my notes default false but perhaps I wrote that down wrong.  Making it an enum seem like reasonable or switching it to upscaleNotAllowed = true